src/cf-base.less

```diff
@@ -140,70 +140,82 @@ body {
     line-height: @base-line-height;
 }
 
-.heading-1( @fs: 34px, @line-height: 44px ) {
+.heading-1( @fs: 34px ) {
     @font-size: @fs;
     .webfont-regular();
-    // Spacing in the design manual is based off of the baseline so this margin
-    // number has been eyeballed to be 30px.
-    margin-bottom: unit( 16px / @font-size, em );
+
+    margin-bottom: unit( 20px / @font-size, em );
     font-size: unit( @font-size / @base-font-size-px, em );
-    line-height: unit( @line-height / @font-size );
+    line-height: 1.25;
 }
 
-.heading-2( @fs: 26px, @line-height: 33px ) {
+.heading-2( @fs: 26px ) {
     @font-size: @fs;
     .webfont-regular();
-    // Spacing in the design manual is based off of the baseline so this margin
-    // number has been eyeballed to be 30px.
-    margin-bottom: unit( 19px / @font-size, em );
+
+    margin-bottom: unit( 20px / @font-size, em );
     font-size: unit( @font-size / @base-font-size-px, em );
-    line-height: unit( @line-height / @font-size );
+    line-height: 1.25;
+
+    p ~ & {
```

Comment:
> The second situation would break even with p + &, so I don't know what the correct answer is here.

That's true. What I never actually worked out was what I would propose :)

One possibility: Set bottom margin on lists, images, figures, blockquotes, tables, etc., such that they do what we want.

Another possibility: Set `margin-top` all the time on raw `hx` elements (well, the `heading-x` mixins, I suppose) and use :first-child to override it to 0.